tests/test_non_ascii_paths/code/makefile

```diff
@@ -0,0 +1,8 @@
+all: c
```

Comment:
(Commenting here because it won't let me comment on a binary file.) Did you mean to check "c" in?